### PR TITLE
Issue #3951: SeparatorWrap add support for method ref

### PR DESCRIPTION
**Pull request by Pedro12909**
Issue https://github.com/checkstyle/checkstyle/issues/3951

### CODE REVIEW

File: src/main/resources/google_checks.xml [hunk 1]
```diff
@@ -91,6 +91,7 @@
         <module name="SeparatorWrap">
             <property name="id" value="SeparatorWrapComma"/>
             <property name="tokens" value="COMMA"/>
+            <property name="tokens" value="METHOD_REF"/>
```
Comment (rnveach) [inline src/main/resources/google_checks.xml]: We need confirmation that the google style wants EOL for this token.
Please look at http://checkstyle.sourceforge.net/reports/google-java-style-20160712.html and make note if it references method references and how they should wrap. You may have to read between the lines with google's style guide.

If it does make reference, point me to the area, if not, I will also confirm, and we will most likely have to remove this from the check and make an override to give a reason why it isn't added.

**Comment by rnveach** [comment]
@Pedro12909 Everything ok?
To make changes to a PR, you don't have to recreate it. You can just amend the current commits and repush the branch.

**Comment by romani** [comment]
Push with "--force"

**Comment by Pedro12909** [comment]
@rnveach I'm still learning how to use UT to test changes, so Im not confident in pushing changes right now. I closed PR because I don't want to be stuck on an issue while others could be fixing it meanwhile

**Comment by rnveach** [comment]
@Pedro12909 If you are releasing this issue for others to work on, you need to make mention of this in issue. Otherwise it looks like you are still working on it.

**Pull request closed without merging**